_summaries/maas2013rectifier_2.md

```diff
@@ -1,19 +1,36 @@
 ---
 layout: summary
-title: Summary
-paper: {{paper_tag}}
+title: Rectifier Nonlinearities Improve Neural Network Acoustic Models, Maas, Hannun, Ng; 2013
+paper: 	{maas2013rectifier}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: sritank
+score: 8/10
 ---
 
 TODO: Summarize the paper:
 * What is the core idea?
+
+DNNs with ReLU and other non-sigmoid activation functions performed better on image recognition tasks. So the paper explores using DNNs as acoustic models and how rectifier non-linearities can improve estimation accuracy of the acoustic model.
```

Comment:
The second sentence is fine, although I would suggest removing the "So" and say "This paper" as the "So" makes the summary seem a bit informal . I also think you should mention something about some of the core terms they used to explain the ReLu DNN performance over the sigmoidal DNN performance (namely sparsity and dispersion) as this is something that is emphasized in the paper.

Reply:
Done